cowamm/profitability/competitor_kpis/sushiswap/sushiswap_kpis_4232588.sql

```diff
@@ -0,0 +1,79 @@
+-- Computes volume, tvl and APR for Sushiswap pools
+-- APR is measured as the fees earned per $ invested, over the last 24 hours, projected over 1 year
+-- Input: blockchain
+
+with
```

Comment:
If this queries are 99% the same (maybe modulo some event topic) we should not duplicate all this code, but instead have the thing that changes a parameter (this way the actual pancake, uni, sushi query become just a `select * from query...` and the whole thing will be much easier to maintain.